dart/lib/src/protocol/sentry_stack_frame.dart

```diff
@@ -0,0 +1,106 @@
+class SentryStackFrame {
+  static final SentryStackFrame asynchronousGapFrameJson = SentryStackFrame()
```

Comment:
if this is internal, we can make it private

Reply:
✅